tests-functional/tests/test_delete_account.py

```diff
@@ -0,0 +1,49 @@
+import copy
+import pytest
+from resources.constants import new_account_data_1
+
+
+@pytest.mark.rpc
+class TestDeleteAccount:
+
+    @pytest.fixture(autouse=True)
+    def setup_backends(self, backend_new_profile):
+        self.account = backend_new_profile("sender")
+        self.account_data = copy.deepcopy(new_account_data_1)
+
+    def test_delete_watch_account(self):
+        self.account_data["type"] = "watch"
+        self.account.accounts_service.add_account(self.account.password, self.account_data)
```

Comment:
@fbarbu15  since you've merged this and I had to rebase my work, I updated this to an empty string in my PR (just to be clear for future devs).

So in case you're adding/removing a watch-only account, the password can be anything, so it can be a real one, but also can be an empty string, cause it's not needed.